client/actions/tests/privileges_test.js

```diff
@@ -36,6 +36,15 @@ describe('privileges', () => {
                     type: PRIVILEGES.ACTIONS.RECEIVE_PRIVILEGES,
                     payload: privileges.privileges,
                 }])
+
+                done()
+            })
+            .catch((error) => {
+                /* eslint-disable no-console */
+                console.log('Unhandled exception: ' + error + '\n' + error.stack)
```

Comment:
Yes, as they are valuable to determining what caused the error.
Otherwise it merely shows an error such as 'event is undefined'

With the console output, we get the method that was last called, i.e. the stacktrace